client/src/components/AuditAdmin/Progress/JurisdictionDetail.tsx

```diff
@@ -125,6 +128,16 @@ const JurisdictionDetail: React.FC<IJurisdictionDetailProps> = ({
             auditSettings={auditSettings}
           />
         )}
+        {lastLoginActivity && (
```

Comment:
What happens if the only login thus far was by a support user logging in as someone else? Will we still render a last login in that case? My gut feeling is that we should filter those out. I think that has to happen on the backend as the frontend only gets the very last login, which may have been a support user but there may have been a real login just before

I'm okay pushing that to a follow-up next week if need be!

Reply:
IIRC, we don't record an activity when a support user logs in as a JM or AA. I can't recall why, but my guess is that we thought it would be too distracting to see in the activity log.